source/reference/minio-mc/mc-ilm-add.rst

```diff
@@ -79,154 +77,153 @@ Parameters
-   limited system resources may delay application of lifecycle management
-   rules. See :ref:`minio-lifecycle-management-scanner` for more information.
+   MinIO uses a scanner process to check objects against all configured lifecycle management rules. 
+   Slow scanning due to high IO workloads or limited system resources may delay application of lifecycle management rules. 
+   See :ref:`minio-lifecycle-management-scanner` for more information.
 
 .. mc-cmd:: --noncurrentversion-transition-days
-   
+   :optional:
+
+   The number of days an object has been non-current (i.e. replaced by a newer version of that same object) after which MinIO marks the object   version as eligible for transition. 
+   MinIO transitions the object to the configured remote tier specified to the :mc-cmd:`~mc ilm add --tier` once the system host datetime passes that calendar date.
+
+   This option has no effect on non-versioned buckets. 
+   Requires specifying :mc-cmd:`~mc ilm add --noncurrentversion-tier`.
+
+   This option has the same behavior as the S3 ``NoncurrentVersionTransition`` action.
 
-   *Optional* The number of days an object has been non-current (i.e. replaced
-   by a newer version of that same object) after which MinIO marks the object
-   version as eligible for transition. MinIO transitions the object to the
-   configured remote storage tier specified to the 
-   :mc-cmd:`~mc ilm add --storage-class` once the system host datetime
-   passes that calendar date.
+   MinIO uses a scanner process to check objects against all configured lifecycle management rules. 
+   Slow scanning due to high IO workloads or limited system resources may delay application of lifecycle management rules. 
+   See :ref:`minio-lifecycle-management-scanner` for more information.
 
-   This option has no effect on non-versioned buckets. Requires specifying
-   :mc-cmd:`~mc ilm add --noncurrentversion-transition-storage-class`.
+.. mc-cmd:: --noncurrentversion-tier
+   :optional:
 
-   This option has the same behavior as the 
-   S3 ``NoncurrentVersionTransition`` action.
+   The remote tier to which MinIO :ref:`transitions noncurrent objects versions <minio-lifecycle-management-tiering>`. 
+   Specify a remote tier created by :mc:`mc admin tier`.
 
-   MinIO uses a scanner process to check objects against all configured
-   lifecycle management rules. Slow scanning due to high IO workloads or
-   limited system resources may delay application of lifecycle management
-   rules. See :ref:`minio-lifecycle-management-scanner` for more information.
+.. mc-cmd:: --newer-noncurrentversions-expiration
+   :optional:
 
-.. mc-cmd:: --noncurrentversion-transition-storage-class
+   The number of non-current object versions to keep as unexpired.
+   If not specified, MinIO marks all non-current object versions as unexpired.
+
+   Use this flag to retain a certain number of past versions of a file in a first in, first out fashion.
```

Comment:
oh, move this up to be above 213. that way our ordering is

- What it is
- How it works
- What happens if you omit it

Reply:
Reordered